js/ui/file_manager/ui.file_manager.item_list.thumbnails.js

```diff
@@ -40,6 +44,8 @@ class FileManagerThumbnailsItemList extends FileManagerItemListBase {
         this.$element().addClass(FILE_MANAGER_THUMBNAILS_ITEM_LIST_CLASS);
         this.$element().append(this._$viewPort);
 
+        const eventName = addNamespace(contextMenuEventName, FILE_MANAGER_THUMBNAILS_CONTEXT_MENU_EVENT);
+        eventsEngine.on(this.$element(), eventName, this._onContextMenu.bind(this));
         eventsEngine.on(this.$element(), "click", this._getItemSelector(), this._onClick.bind(this));
```

Comment:
Add namespace?

Reply:
OK. Now the namespace has been added to the "click" event too.